guidelines/index.html

```diff
@@ -4198,7 +4198,74 @@ <h2 id="x7-input-purposes-for-user-interface-components"><span class="secno">7.
 		<li><strong>impp</strong> - インスタントメッセージングプロトコルのエンドポイントを表す URL (例えば、"<strong>aim:goim?screenname=example</strong>"又は <strong>xmpp:fred@example.net</strong>")</li>
 	</ul><!--OddPage-->
 	
-</section>
+      </section>
+
+      <section class="appendix" id="changelog">
+	<h2 id="a-change-log"><bdi class="secno">A. </bdi>変更点<span class="permalink"><a class="self-link" href="#changelog" aria-label="Permalink for Appendix A."><span>§</span></a></span></h2>
+	
+         <p>このセクションでは、WCAG 2.1 が <abbr title="World Wide Web Consortium">W3C</abbr> 勧告として発行されてからの変更点を示す。これらの変更点は、<a href="https://www.w3.org/WAI/WCAG21/errata/">正誤表</a>としても記録されている。
```

Comment:
Errataは訳さなくていいかもしれません。これは"Errata"という名前の文書を指していると解釈でき、文書名は訳さない方針であるためです。

とはいえ Errataくらいは訳してもいいんじゃないか、という感じもします。
https://github.com/waic/wcag21/pull/1836 でも同じ理由で errata のままにしていますので、もし訳すならそっちも訳すことになります。

```suggestion
         <p>このセクションでは、WCAG 2.1 が <abbr title="World Wide Web Consortium">W3C</abbr> 勧告として発行されてからの変更点を示す。これらの変更点は、<a href="https://www.w3.org/WAI/WCAG21/errata/">Errata</a>としても記録されている。
```

Reply:
「Errata」は訳さずママ、了解です。他にも影響範囲があるので、無理に訳さなくてよいかと思います。